guide/yaml/winrm/index.md

```diff
@@ -515,11 +515,9 @@ Known Limitations
 
 ### Use of Unencrypted HTTP
 
-Brooklyn is currently using unencrypted HTTP for WinRM communication. This means that the login credentials will be
-transmitted in clear text.
+By default Brooklyn is currently using unencrypted HTTP for WinRM communication. It does not support encrypted HTTP for WinRM.
```

Comment:
I think this should be - Currently, Brooklyn does not support encrypted HTTP communication for WinRM.